### PR TITLE
[ISSUE #222]When onChange is NEW, eventMesh occurred NullPointException

### PR DESCRIPTION
**Pull request by xwm1992**

### CODE REVIEW

**Review by wqliang — APPROVED** [review]
LGTM

File: docker/eventmesh-defibus/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM centos7-jdk8:latest
```
Comment (keranbingaa) [inline docker/eventmesh-defibus/Dockerfile:1]: This image should give some relevant instructions.

**Review by keranbingaa — APPROVED** [review]
LGTM

File: docker/eventmesh-defibus/Dockerfile (hunk 1 above)
Comment (xwm1992) [reply docker/eventmesh-defibus/Dockerfile:1]: OK, I'll supply readmes for Dockerfiles.

**Pull request merged**